src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -110,6 +110,8 @@ def create(style_dataset, content_dataset, style_feature=None,
         raise _ToolkitError("content_dataset SFrame cannot be empty")
     if(batch_size < 1):
         raise _ToolkitError("'batch_size' must be greater than or equal to 1")
+    if max_iterations is not None and (type(max_iterations)==str or type(max_iterations)==float or max_iterations<0):
```

Comment:
It's probably cleaner to write this line as:
```python
if max_iterations is not None and (isinstance(max_iterations, (str, float)) or max_iterations < 0):
```
But I think the logic should actually be:
```python
if max_iterations is not None and (not isinstance(max_iterations, int) or max_iterations < 0):
```

Reply:
Thanks! This reads a lot cleaner